Resources/Prototypes/Entities/Objects/Devices/pda.yml

```
@@ -41,6 +42,22 @@
     mask: /Textures/Effects/LightMasks/cone.png
     autoRot: true
   - type: Ringer
+  - type: DeviceNetwork
+    deviceNetId: Wireless
+    receiveFrequencyId: PDA
+    prefix: device-address-prefix-console
+  - type: WirelessNetworkConnection
+    range: 500
+  - type: CartridgeLoader
+    uiKey: enum.PDAUiKey.Key
+    cartridgeSlot:
+      priority: -1
+      name: Cartridge
+      ejectSound: /Audio/Machines/id_swipe.ogg
+      insertSound: /Audio/Weapons/Guns/MagIn/batrifle_magin.ogg
+      whitelist:
+        components:
+          - Cartridge
   - type: ActivatableUI
     key: enum.PDAUiKey.Key
     singleUser: true
```

```suggestion
    singleUser: true
    closeOnHandDeselect: false
```
Might be a nice time to add this to the ActivationUI now that it's a possibility. This makes it so you can switch the active hand and the PDA UI would remain up, which would likely be useful for keeping an eye on whatever you are doing on your PDA while multitasking